helpers/isCommentLikedByUser.js

```diff
@@ -0,0 +1,19 @@
+import { CommentLikes } from '../models';
+
+/**
+ * @description This function checks if a function has been liked by a user
```

Comment:
Correct the mistake in your description js.docs

Reply:
Updated, thanks